stix_shifter_modules/guardium/stix_translation/json/stix_2_1/qsearch_from_stix_map.json

```diff
@@ -0,0 +1,46 @@
+{
+  "x-ibm-finding": {
+      "fields": {
+        "finding_type": ["datacategory"],
+        "start": ["startTime"],
+        "end": ["endTime"],
+        "database_name": ["Database"],
+        "dst_device": ["Database"]
+      }
+    },
+    "ipv4-addr": {
+      "fields": {
+        "value": ["ClientIP", "Server"],
+        "src_ip": ["ClientIP"],
+        "dst_ip": ["Server"]
+      }
+    },
+    "ipv6-addr": {
+      "fields": {
+        "value": ["ClientIP", "Server"],
+        "src_ip": ["ClientIP"],
+        "dst_ip": ["Server"]
+      }
+    },
+    "network-traffic": {
+      "fields": {
+        "dst_ref.value": ["Server"],
+        "src_ref.value":["ClientIP"]
+      }
+    },
+     "user-account": {
+      "fields": {
+        "value": ["DB User", "OS User"],
+        "db_user": ["DB User"],
+        "os_user": ["OS User"],
+        "user_id": ["DB User"],
```

Comment:
if you have multiple fields that maps to user_id then just specify them in the list

Reply:
Are you refering to standard user object and  can it be taken for next release?